runtime/vm/createramclass.cpp

```diff
@@ -1696,10 +1696,19 @@ loadSuperClassAndInterfaces(J9VMThread *vmThread, J9ClassLoader *classLoader, J9
 
 #if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
 /**
- * Attempts to pre-load classes of fields with Q signatures and no static
- * access modifier set. Also records largest field type in valueTypeFlags.
- * 0 for single 1 for ref and 2 for double. Records total number of
+ * This method has three main functions:
+ * 1. Attempts to pre-load classes of fields with Q signatures and no 
+ * static access modifier set.
+ * 
+ * 2. Records largest field type in valueTypeFlags.
+ * 0 for single, 1 for ref and 2 for double. Records total number of
```

Comment:
The original version of that comment is out of date. Basically it should say if the field is double then it ors in `J9ClassLargestAlignmentConstraintDouble`, ... similar for ref ... (you can fill in the details) ... nothing for single. Later on, only one of the 3 options will be selected. (Double > ref > single). Something like that.